cns/service/main.go

```diff
@@ -200,15 +258,18 @@ func main() {
 	logLevel := acn.GetArg(acn.OptLogLevel).(int)
 	logTarget := acn.GetArg(acn.OptLogTarget).(int)
 	logDirectory := acn.GetArg(acn.OptLogLocation).(string)
-	ipamQueryUrl, _ := acn.GetArg(acn.OptIpamQueryUrl).(string)
-	ipamQueryInterval, _ := acn.GetArg(acn.OptIpamQueryInterval).(int)
+	ipamQueryUrl := acn.GetArg(acn.OptIpamQueryUrl).(string)
+	ipamQueryInterval := acn.GetArg(acn.OptIpamQueryInterval).(int)
 	startCNM := acn.GetArg(acn.OptStartAzureCNM).(bool)
 	vers := acn.GetArg(acn.OptVersion).(bool)
 	createDefaultExtNetworkType := acn.GetArg(acn.OptCreateDefaultExtNetworkType).(string)
 	telemetryEnabled := acn.GetArg(acn.OptTelemetry).(bool)
 	httpConnectionTimeout := acn.GetArg(acn.OptHttpConnectionTimeout).(int)
 	httpResponseHeaderTimeout := acn.GetArg(acn.OptHttpResponseHeaderTimeout).(int)
 	storeFileLocation := acn.GetArg(acn.OptStoreFileLocation).(string)
+	privateEndpoint := acn.GetArg(acn.OptPrivateEndpoint).(string)
```

Comment:
can you rename privateEndpoint to also include dnc since this is the dnc endpoint?

Reply:
what do you mean, I think it's aptly named in this instance

Reply:
isn't that PE specific to dnc? I just thought having dnc in the name would clarify that.

Reply:
it goes from CNS->PE->PL->ILB->DNC, from CNS POV it just talks to PE as if it's DNC

Reply:
i think ashvin point here is it would be more readable ..by looking at code tomorrow anyone can get some sense this PE is intended for dnc communication